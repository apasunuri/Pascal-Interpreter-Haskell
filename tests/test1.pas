program test1;

var
    x, y: real;

begin
    if(5 <> 7) then
        begin
            writeln(5 / 7);
            writeln('Valid Statement');
        end
    else
        writeln('Not a Valid Statement');
    if(5 > 7) then
        writeln(sqrt(5 / 7))
    else
        writeln('Not a Valid Statement');
end.