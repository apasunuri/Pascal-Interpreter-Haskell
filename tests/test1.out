0.71428573
Valid Statement
Not a Valid Statement